cmd/proxy/main.go

```diff
@@ -94,11 +103,31 @@ VERSION:
 		Usage: "Starts the proxy as a rosetta server",
 	}
 
+	// logLevel defines the logger level
+	logLevel = cli.StringFlag{
+		Name: "log-level",
+		Usage: "This flag specifies the logger `level(s)`. It can contain multiple comma-separated value. For example" +
+			", if set to *:INFO the logs for all packages will have the INFO level. However, if set to *:INFO,api:DEBUG" +
+			" the logs for all packages will have the INFO level, excepting the api package which will receive a DEBUG" +
+			" log level.",
+		Value: "*:" + logger.LogInfo.String(),
+	}
+	//logFile is used when the log output needs to be logged in a file
+	logSaveFile = cli.BoolFlag{
+		Name:  "log-save",
+		Usage: "Boolean option for enabling log saving. If set, it will automatically save all the logs into a file.",
+	}
+	// workingDirectory defines a flag for the path for the working directory.
```

Comment:
you can drop this parameter and keep only the logs directory
anyway, if you keep it, at least change the `Usage`

Reply:
done.